inference-engine/tests/functional/plugin/shared/src/single_layer_tests/loop.cpp

```diff
@@ -161,7 +161,7 @@ namespace LayerTestsDefinitions {
 
     void StaticShapeLoopTest::SetUp() {
         SKIP_IF_CURRENT_TEST_IS_DISABLED()
-        SetRefMode(LayerTestsUtils::IE);
+        SetRefMode(LayerTestsUtils::INTERPRETER);
```

Comment:
I think, we can remove this line (It's default option)

Reply:
done